budgetportal/models.py

```diff
@@ -1412,6 +1416,148 @@ class PackageWithoutGroupException(Exception):
     pass
 
 
+class InfrastructureProject():
+    """ Represents an infrastructure project stored in CKAN """
+
+    def __init__(self, **kwargs):
+        self.records = kwargs.get('records')
+        if self.records:
+            self.name = self.records[0]['Project name']
+            self.stage = self.records[0]['Current project stage']
+            self.department_name = self.records[0]['Department']
+            self.description = self.records[0]['Project description']
+            self.total_budget = float(self.records[0]['Total project cost'])
+            self.nature_of_investment = self.records[0]['Nature of investment']
+            self.infrastructure_type = self.records[0]['Infrastructure type']
+            self.gps_codes = self.records[0]['GPS code']
+        self.coordinates = []
+        self.provinces = []
+        self.expenditure = []
+
+    @classmethod
+    def get_dataset(cls):
+        """ Return the first dataset in the Infrastructure Projects group. """
+        query = {
+            'q': '',
+            'fq': ('+organization:"national-treasury"'
+                   '+vocab_spheres:"national"'
+                   '+groups:"infrastructure-projects"'),
+            'rows': 1,
+        }
+        response = ckan.action.package_search(**query)
+        logger.info(
+            "query %s\nreturned %d results",
+            pformat(query),
+            len(response['results'])
+        )
+        if response['results']:
```

Comment:
looks like this will crash in production until we upload this data? But we need to be able to deploy this and just have the data appear in the static site once it gets uploaded to ckan